src/ExpressiveInstaller/config.php

```diff
@@ -2,16 +2,16 @@
 
 return [
     'packages' => [
-        'aura/di'                                        => '^3.0',
-        'filp/whoops'                                    => '^2.0',
+        'aura/di'                                        => '^3.2',
+        'filp/whoops'                                    => '^2.1.7',
         'xtreamwayz/pimple-container-interop'            => '^1.0',
         'zendframework/zend-expressive-aurarouter'       => '^2.0',
         'zendframework/zend-expressive-fastroute'        => '^2.0',
-        'zendframework/zend-expressive-platesrenderer'   => '^1.2',
-        'zendframework/zend-expressive-twigrenderer'     => '^1.2.1',
-        'zendframework/zend-expressive-zendrouter'       => '^2.0',
-        'zendframework/zend-expressive-zendviewrenderer' => '^1.2.1',
-        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.2',
+        'zendframework/zend-expressive-platesrenderer'   => '^1.2.1',
+        'zendframework/zend-expressive-twigrenderer'     => '^1.3',
+        'zendframework/zend-expressive-zendrouter'       => '^2.0.1',
+        'zendframework/zend-expressive-zendviewrenderer' => '^1.3',
+        'zendframework/zend-servicemanager'              => '^3.3 || ^2.7.8',
```

Comment:
That depends if there are still zend components that are on the 2.x versions. If every component accepts 3.x it can be removed. If not you will get composer errors about constrain clashes.